wheel.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+arrow_dir=$1
+build_dir=$2
+
+export ARROW_FLIGHT=OFF
+export ARROW_JEMALLOC=OFF
+export ARROW_SIMD_LEVEL=NONE
+export BUILD_PREFIX=$build_dir
+export CONFIG_PATH=/dev/null
+export MACOSX_DEPLOYMENT_TARGET=11.0
+export MB_PYTHON_VERSION=3.9
+export PLAT=arm64
```

Comment:
The produced wheel has universal2 platform tag, though my original plan is to provide a pure arm64 wheel in the first iteration.

Reply:
Tried `export ARCHFLAGS=" -arch arm64"`, but still.